elements/lisk-chain/test/unit/chain.spec.ts

```diff
@@ -35,8 +35,7 @@ describe('chain', () => {
 	const constants = {
 		blockReceiptTimeout: 20,
 		loadPerIteration: 1000,
-		maxPayloadLength: 1024 * 1024,
-		maxTransactionsPerBlock: 25,
+		maxPayloadLength: 15 * 1024,
```

Comment:
can we use this from the constants or config? so that if SDK user wants to configure the payload size the test is intact. same comment for rest of the test below.

Reply:
1) This commit updates the default value for `maxPayloadLength` from `1 Mb` to `15 kB`  and it is already configurable as was before I think -> 
https://github.com/LiskHQ/lisk-sdk/pull/4884/commits/26c898749d900f6a5b8d42e217861d8f08e78035

2) The issue only says to change it in `elements/lisk-chain` and only to remove the check condition and from the constructor. Also, we still want to accept it in the block header? @shuse2

Reply:
ok, in that case, my comment is invalid, what about the comment on point 2, reference to `maxTransactionsPerBlock` exists in code. We dont remove them?

Reply:
For point 2, only the check should be removed, I reverted the changes where I removed it from the types in lisk-chain library. This is what the issue says and we want to keep it in the `blockHeader`